backend/main/chapters/c11_tic_tac_toe_project.py

```diff
@@ -589,3 +589,346 @@ def generate_inputs(cls):
 Now we have the code to determine a winning state on the board.
 Next we will tackle the problem of displaying the board on the screen.
 """
+
+
+class NewlinesAndFormatBoard(Page):
+    title = "Introducing the newline character, and `format_board`"
+
+    class one_way_to_print_board(VerbatimStep):
+        """
+Next we want to tackle the problem of displaying the tic-tac-toe board. Here's one way to do this:
+
+    __copyable__
+    def print_board(board):
+        for row in board:
+            print("".join(row))
+
+    print_board([
+        ['X', 'O', 'X'],
+        [' ', 'O', 'O'],
+        [' ', 'X', ' ']
+    ])
+
+(What's `"".join`? Google it!)
+        """
+
+        program_in_text = False
```

Comment:
why?

Reply:
Hmm... I think I was under the false impression that `__program_indented__` has to be the very last thing in the class string.

Reply:
Should be fixed in the next commit.